lib/views/pages/login_signup/register_form.dart

```diff
@@ -487,4 +487,4 @@ class RegisterFormState extends State<RegisterForm> {
       _obscureText = !_obscureText;
     });
   }
-}
\ No newline at end of file
+}
```

Comment:
What have you changed here ? Is it required ?

Reply:
The **change doesn't break anything** I had to merge my local repo with my fork and during this this was added.
But the main commits are ID: d5a925c & abf2468 which can be cherry-picked I guess